nodejs/week1/avg.js

```diff
@@ -0,0 +1,10 @@
+const numbers = process.argv.slice(2);
+
+if (numbers.length > 0) {
+    const sum = numbers.reduce((a, b) => +a + +b);
+
+    numbers && isNaN(sum) ? console.log('Input should be a number') : console.log( sum / numbers.length );        
```

Comment:
Overall, this is a good solution, you did a good job. However, I would prefer to check if the values on my reduce function are numbers before, this could help prevent errors in your future work.

Reply:
That’s true. I will modify it using find method.

Reply:
Hi Natalia,

I modified the code like below,

const numbers = process.argv.slice(2);

if (numbers.length > 0) {
    const isNotNumber = numbers.find(each => isNaN(+each) === true);
    isNotNumber
            ? console.log('Input should be a number')
            : console.log( numbers.reduce((a, b) => +a + +b) / numbers.length );
}
else {
    console.log('Enter numbers to get the average')
}

Reply:
I dont see the commit where you did this change. You should have committed and push your changes to this branch.